src/Illuminate/Database/SQLiteConnection.php

```diff
@@ -25,6 +25,15 @@ class SQLiteConnection extends Connection
      */
     public function __construct($pdo, $database = '', $tablePrefix = '', array $config = [])
     {
+        // check that if sqlite file currently not exists, create it
+        if (! is_file($database) && $database !== ':memory:') {
```

Comment:
Consider reversing the conditions, so it can bail right away if it is a `:memory` database.

`if ($database !== ':memory:' && ! is_file($database)) {`

Reply:
Thanks. done.